beacon-chain/node/node.go

```diff
@@ -970,3 +973,7 @@ func (b *BeaconNode) registerValidatorMonitorService() error {
 	}
 	return b.services.RegisterService(svc)
 }
+
+func hasNetworkFlag(cliCtx *cli.Context) bool {
+	return cliCtx.IsSet(features.PraterTestnet.Name) || cliCtx.IsSet(features.Mainnet.Name) || cliCtx.IsSet(features.RopstenTestnet.Name)
```

Comment:
@saolyn im concerned we'll have to update this function every time we add a new network flag. It's one more thing for all of us to remember. Maybe something we could do is in our features package we could have a network flags slice and then just if anything from that slice is set by iterating over it here? Also, in our main.go or usage.go files we could just do
```go
Flags: []cli.Flag{
  someFlag,
  otherFlag,
  networkFlags...,
}
```
Instead of specifying RopstenFlag, MainnetFlag, PraterFlag

Reply:
I was struggling to find a way to add it to `main.go` or `usage.go` so I settled on creating a `NetworkFlags` group in `features`. Was this what you had in mind?

Reply:
I think that's a good path!

Reply:
@terencechain and @rauljordan I fixed the conflicts, can you please confirm if the way I am handling the network flags is as expected.